Monero_Payments.php

```diff
@@ -88,24 +88,21 @@ public function get_transfers(){
     }
      
      public function view_key(){
-    $query_key = array('key_type' => 'view_key');
-    $query_key_method = $this->_run('query_key', $query_key);
-    $this->_print($query_key_method);
+        $query_key = array('key_type' => 'view_key');
+        $query_key_method = $this->_run('query_key', $query_key);
+        $this->_print($query_key_method);
      }
      
+        /* A payment id can be passed as a string
+           A random payment id will be generatd if one is not given */
      public function make_integrated_address($payment_id){
-         if(isset($payment_id)){
-             $integrate_address_parameters = array('payment_id' => '');
```

Comment:
I have implemented this because, if we don't pass a payment_id,  wallet will try to generate a new one